### ISSUE TITLE
DataTable error with Angular 2.4.1

### ISSUE THREAD

**Issue by ctrl-brk** [primefaces/primeng#1699]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**
In Antular 2.4.1 I'm getting the following error from DataTable
`./DataTable class DataTable - inline template:157:128 caused by: trackBy must be a function, but received undefined`
There's no error with Angular 2.4.0

**Expected behavior**
No error

**Please tell us about your environment:**

* **Angular version:** 2.4.1
* **Angular CLI version:** 1.0.0-beta.24

* **PrimeNG version:** 1.1.1

* **Browser:**
 Chrome 55.0.2883.87 m

**Comment by ctrl-brk** [comment]
I should say that PrimeNG 1.1.2 doesn't have this issue.
And need to remind that column headers still disappear when sorting, so It's still necessary to hack the datatable.js file.

**Comment by Thavarajan** [comment]
workaround for this issue is
create a function with signgle parameter
and assign the function  to the rowTrackBy property of the datatable object

**Comment by Mrtcndkn** [comment]
We fixed it for 1.1.2 but it's temporary.
